framework/src/modules/interoperability/base_interoperable_method.ts

```diff
@@ -18,11 +18,15 @@ import {
 	BeforeSendCCMsgMethodContext,
 	BeforeRecoverCCMsgMethodContext,
 	RecoverCCMsgMethodContext,
+	CrossChainMessageContext,
 } from './types';
 
 export abstract class BaseInteroperableMethod extends BaseMethod {
 	public beforeApplyCCM?(ctx: BeforeApplyCCMsgMethodContext): Promise<void>;
 	public beforeSendCCM?(ctx: BeforeSendCCMsgMethodContext): Promise<void>;
 	public beforeRecoverCCM?(ctx: BeforeRecoverCCMsgMethodContext): Promise<void>;
 	public recover?(ctx: RecoverCCMsgMethodContext): Promise<void>;
+	public verifyCrossChainMessage?(ctx: CrossChainMessageContext): Promise<void>;
```

Comment:
There was a discussion with @ishantiw in which we thought `BaseCCMethod` was better for this class with file name `base_cc_method`. It is named so in my PR

Reply:
sounds good to me! I think you can update it in ur PR 🙏